### PR TITLE
Fwdpy 0.0.4pre1

### PR DESCRIPTION
**Pull request by molpopgen**
* [ X] I have read the guidelines above.
* [ X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by molpopgen** [comment]
@bioconda/core  Ready for review.

File: recipes/fwdpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - libsequence
+    - pandas
+    - numpy
+    - fwdpp >=0.5.4
+    - gsl {{CONDA_GSL}}*
+
+  run:
+    - python
+    - numpy
+    - numpy
```
Comment (bgruening) [inline recipes/fwdpy/meta.yaml]: two times numpy, you probably also want to build it against multiple numpy versions?

**Comment by molpopgen** [comment]
@bioconda/core OK, the changes are made.  I changed it to require minimum pandas/numpy versions.  Nothing special in terms of functionality is needed for the build, and so a relatively recent version should be more than ok.

**Comment by bgruening** [comment]
@molpopgen just to double check, you are not compiling against the numpy API?

**Comment by molpopgen** [comment]
No. It is just a build Requiem in requires.txt.

**Comment by bgruening** [comment]
Ok, thanks!

**Pull request merged**